learning-examples/bonding-curve-progress/get_bonding_curve_status.py

```
@@ -37,18 +40,42 @@ class BondingCurveState:
         "complete" / Flag,
+        "creator" / Bytes(32),  # Added new creator field - 32 bytes for Pubkey
     )
 
     def __init__(self, data: bytes) -> None:
-        parsed = self._STRUCT.parse(data[8:])
-        self.__dict__.update(parsed)
+        """Parse bonding curve data."""
+        if data[:8] != EXPECTED_DISCRIMINATOR:
+            raise ValueError("Invalid curve state discriminator")
+
+        if len(data) < 150:
```

Ruff (0.8.2)

Magic value used in comparison, consider replacing `150` with a constant variable

(PLR2004)